modules/rtticketedit.php

```diff
@@ -133,6 +133,14 @@
                 $SESSION->redirect('?m=rtqueueview'
                     . ($SESSION->is_set('backid') ? '#' . $SESSION->get('backid') : ''));
                 break;
+            case 'resetpriority':
+                $ticket = $LMS->GetTicketContents($id);
+                if ($ticket['priority'] != RT_PRIORITY_NORMAL) {
+                    $LMS->TicketChange($id, array('priority' => RT_PRIORITY_NORMAL));
+                }
+                $SESSION->redirect('?m=rtqueueview'
+                    . ($SESSION->is_set('backid') ? '#' . $SESSION->get('backid') : ''));
+                break;
```

Comment:
Gdy zgłoszenie ma przypisanego weryfikatora lub ma status _weryfikowany_ to również można wyczyścić priorytet zgłoszenia?

Reply:
Tak - ponieważ praca może zostać wykonana a czynność weryfikacji nie jest już tak priorytetowa.